sales/src/errors.rs

```diff
@@ -0,0 +1,29 @@
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ContractError {
+    #[error("{0}")]
+    Std(#[from] StdError),
+
+    #[error("Unauthorized")]
+    Unauthorized,
+
+    #[error("Ongoing Primary Sale Error")]
```

Comment:
We can remove "Error" from all of the error strings in this file (e.g. "Ongoing Primary Sale")

Reply:
okay, sounds more elegant. Was just sticking to a pattern